hotdoc/utils/utils.py

```diff
@@ -160,10 +167,30 @@ def __get_extra_extension_classes(paths):
 
     return extra_classes
 
+def __load_extra_extension_moudles(paths: T.List[str]) -> T.List[T.Type['Extension']]:
+    """
+    Banana banana
+    """
+    extra_classes = []
+    for p in [Path(x) for x in paths]:
+        if not p.exists() or not p.is_file():
+            info(f'Extension {p} does not exist or is not a file (skipping)')
+            continue
+        try:
+            spec = importlib.util.spec_from_file_location(p.name.replace('.py', ''), p)
```

Comment:
bit funny, what's `p.name`? The base name? I'm not that familiar with pathlib, is there a way to get to the extension?

Reply:
Yes, it is the basename. Ther are a bunch of other functions for the stem (which I switched to because it is much cleaner) as well: https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.name

In general, I think it is much cleaner to deal with dedicated objects for paths instead of raw strings :)

Reply:
agreed, pathlib was still pretty new and I wasn't familiar with it then, `typing` as well is a great addition I haven't yet used enough, I'm mostly doing C / rust these days. Anyway, stem does look nicer :)